apps/ftu/js/late_customization.js

```diff
@@ -205,19 +220,18 @@
       });
     },
 
-    getOperatorInfo: function() {
+    getInfoFromMobileConnection: function() {
       var connections = navigator.mozMobileConnections;
-      for (var i=0, conn; i<connections.length; i++) {
-        conn = connections[i];
-        if(conn && conn.voice && conn.voice.connected) {
-          var operatorInfos = MobileOperator.userFacingInfo(conn);
-          if (operatorInfos.operator) {
-            operatorInfos.mcc = conn.voice.network.mcc;
-            operatorInfos.mnc = conn.voice.network.mnc;
-            if (!operatorInfos.carrier) {
-              operatorInfos.carrier = operatorInfos.operator;
+      if (connections) {
```

Comment:
Don't fear the indents! Usually I'm an early returner but with a couple conditions and returns in there already, the if seemed clearer.